### ISSUE TITLE
Convert density value to SI system

### ISSUE THREAD

**Issue by ghost** [ueshita/godot-floatable-body#2]
The density is 100 by default. 100 what? Clearly it isn't correct for water as a 1 m³ box stops floating at around 12 kg. Wouldn't it be good to have default density at 1 kg/dm³ so that the default box can't float if it's heavier than 1000 kg? That would make it way easier for anyone using this addon to set the density and object mass.

**Comment by ueshita** [comment]
You are right that the default value for Density is 100, which is strange.
I think it is nice and simple that the default value is 1.0.
Also, there were various errors in the buoyancy process.
I will make various corrections from now on.
